### PR TITLE
[DDW-1112] Implement new "Terms of use" for the "Rewards" network

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR adds new Terms of use for the Rewards network.

## Todos

- [x] Add Japanese translation of the new "Terms of use"

## Screenshots

EN
EN
JP
JP

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/messages/GGKFXSKC6)
- [ ] Check if the "Terms of use" match the document: https://docs.google.com/document/d/1UcmuNzue-4br1yvevJI--lJ_Q5yE4iZH4SKOhs0wT6s/edit#

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/i18n/locales/terms-of-use/itn-rewards-v1/en-US.md [hunk 1]
```diff
@@ -1,25 +1,57 @@
...
 
-11. The term of these Terms will begin on the Effective Date and will continue until we end the Incentivized Testnet. You understand and agree that we may change, withdraw, terminate your access to, testing of and/or use of, or discontinue the Incentivized Testnet (or any portion thereof) at any time and in our sole discretion. You may also end your participation in the Incentivized Testnet at any time and any Rewards gained up to that time can still be transferred to your Daedalus or Yoroi mainet wallet once the Incentivized Testnet is discontinued.
+11. The term of these Terms will begin on the Effective Date and will continue until we end the Incentivized Testnet. You understand and agree that we may change, withdraw, terminate your access to, testing of and/or use of, or discontinue the Incentivized Testnet (or any portion thereof) at any time and in our sole discretion. You may also end your participation in the Incentivized Testnet at any time and any Rewards gained up to that time will still be transferred to your Daedalus or Yoroi mainet wallet once the Incentivized Testnet is discontinued.
+
+<hr />
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/terms-of-use/itn-rewards-v1/en-US.md]: @yakovkaravelov I don't think this is a valid Markdown element.

File: source/renderer/app/i18n/locales/terms-of-use/itn-rewards-v1/en-US.md [hunk 2]
```diff
@@ -1,25 +1,57 @@
...
+
+# IOHK Testnet Stake Pool Staking Terms of Service
+
+Input Output HK Ltd (&quot;IOHK&quot;), by and through <u>http://staking.cardano.org</u> (the &quot;Site&quot;) and as part of the IOHK Incentivised Testnet program (&quot;Incentivized Testnet&quot;), provides a staking service through various stake pools which allows Ada holders to stake the rights associated with their Ada holdings in exchange for a reward equivalent to a percentage of the net proceeds generated by IOHK from staking (&quot;Service&quot;). These Terms of Service (&quot;Terms of Service&quot; or &quot;Terms&quot;) apply to any person participating in the Incentivised Testnet (&quot;User&quot;).
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/terms-of-use/itn-rewards-v1/en-US.md]: @yakovkaravelov `<u>`  is not a valid Markdown syntax :(

**Comment by yakovkaravelov** [comment]
Waiting for Darko's update to receive newly updated terms of use document.

**Comment by a-rukin** [comment]
image
eeemm... @yakovkaravelov why do we have horizontal separator in this place? 
It should only separate two different terms of smth

**Comment by a-rukin** [comment]
image
by design top margin should be 22px, bottom - 11px

**Comment by a-rukin** [comment]
I'd also use --font-light instead of --font-regular (for the regular small text) 
and use --font-medium (without font-weight:bold) for strong/bold small text parts
it helps readability from my pov

**Comment by a-rukin** [comment]
image
section 1.5 is the only one with one more list in the doc, rest lists are inlined 
@darko-mijic what should we do in this case?

**Comment by a-rukin** [comment]
image
We are still using "Terms of use" (but it is now Terms of service)  in the side bar (and for checkbox in first launch setup step)
cc @darko-mijic

**Comment by a-rukin** [comment]
image
In storybook there is a duplication of numbering

File: source/renderer/app/i18n/locales/en-US.json [hunk 3]
```diff
@@ -250,7 +250,7 @@
   "settings.menu.display.link.label": "Themes",
   "settings.menu.general.link.label": "General",
   "settings.menu.support.link.label": "Support",
-  "settings.menu.termsOfUse.link.label": "Terms of use",
+  "settings.menu.termsOfUse.link.label": "Terms of service",
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/en-US.json]: @yakovkaravelov please update this key for Japanese translation too: https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/i18n/locales/ja-JP.json#L253

File: source/renderer/app/i18n/locales/en-US.json [hunk 4]
```diff
@@ -236,8 +236,8 @@
   "profile.settings.numberSelect.label": "Number format",
   "profile.settings.submitLabel": "Continue",
   "profile.settings.timeSelect.label": "Time format",
-  "profile.termsOfUse.checkboxLabel": "I agree with terms of use",
-  "profile.termsOfUse.checkboxLabelWithDisclaimer": "I agree with terms of use",
+  "profile.termsOfUse.checkboxLabel": "I agree with terms of service",
```
Comment (nikolaglumac) [inline source/renderer/app/i18n/locales/en-US.json]: @yakovkaravelov please update this key for Japanese translation too: https://github.com/input-output-hk/daedalus/blob/v2-integration/source/renderer/app/i18n/locales/ja-JP.json#L239

**Pull request merged**